person.rb

```diff
@@ -1,15 +1,17 @@
+require 'pry'
 require './corrector'
 
 class Person
   attr_accessor :name, :age, :permission
-  attr_reader :id, :corrector
+  attr_reader :id, :corrector, :rented
 
   def initialize(age, name, permission: true)
     @id = Random.rand(1..1000)
     @name = name
     @corrector = Corrector.new(@name)
     @age = age
     @permission = permission
+    @rented = []
```

Comment:
- [ ] As it is a  requirement for this project, please a method for adding rentals, `the person itself should be sent as a parameter to create the new rental achieving the 'has-many' association`. Checkout this [link](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship) and kindly fix that.